veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,253 @@
+# VEP #21: Seamless TCP migration and vhost-user support with passt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, linking to VEP directory in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+Kubevirt users can benefit from seamless live migration of VMs without disconnection of TCP sessions, while using `passt`.
+This functionality has recently been introduced by `passt` alongside support of `vhost-user` protocol, which increases network throughput. 
+This proposal outlines the integration plan to be implemented in Kubevirt. 
+
+## Motivation
+
+Currently, KubeVirt does not support all Primary Network requirements expected by users:
+- Seamless network connectivity on migration
+- Observability/Service Mesh support
+- High throughput (on par with tap)
+
+Bridge binding does not support observability or service meshes, Masquerade hides the real IP from the user, and passt currently lacks seamless migration and tap-level performance.
```

Comment:
- the "bridge" binding
- the "masquerade" binding
- IP address
- passt didn't support seamless migration and throughput comparable to a tap interface until recently ("currently" implies something in the present, not in the recent past)